autotst/species.py

```diff
@@ -502,6 +519,44 @@ def get_torsions(self):
 
                 torsion_tup = (atom0.GetIdx(), atom1.GetIdx(),
                                atom2.GetIdx(), atom3.GetIdx())
+                description = 'hindered'
+
+                endpoints = check_angles(torsion_tup)
+                center_atoms = []
+   
+                while len(endpoints) > 0:
+                    
+                    reached_the_end = [False for i in range(len(endpoints))]
+                    for i,index in enumerate(endpoints):
+                        if index == 0:
+                            vertex = torsion_tup[1]
+                        else:
+                            vertex = torsion_tup[2]
+                        ID = torsion_tup[index]
```

Comment:
Maybe rename ID as atom_id, I'm personally not a big fan of all caps variables unless they are constants. Here's a decent [style guide](https://realpython.com/python-pep8/)